common/views/themes/config.ts

```diff
@@ -95,7 +95,7 @@ export const colors = {
   orange: { base: '#e87500', dark: '', light: '' },
   yellow: { base: '#ffce3c', dark: '', light: '#fff5d8' },
   brown: { base: '#815e48', dark: '', light: '' },
-  cream: { base: '#f0ede3', dark: '', light: '#fbfaf4' },
+  cream: { base: '#f0ede3', dark: '#d9d8d0', light: '#fbfaf4' },
```

Comment:
cream dark exists in the new palette but not the existing one. I needed it for this page, so had to add it to people without the toggle wouldn't just see a transparent background. As I'm working on implementing the new one ASAP, I hope this is alright with everyone 👍